_pages/support.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: article
+title: "Algorithmia Support"
+
+show_related: false
+excerpt: "Support tiers"
+image:
+    teaser: /icons/Algorithmia_FAQs.png
+---
+
+Algorithmia provides distinct levels of support depending on your subscription. Send us an email support request to [support@algorithmia.com](mailto:support@algorithmia.com). Please use your corporate domain email addresses if emailing us directly. 
+
+## Algorithmia Teams Pro tier
+
+A value-add feature of the Teams Pro subscription is access to our Technical Support team for any platform and administrative issues as well as user-, data science-, and model-related questions. 
+
+#### Algorithmia offers several options to contact support
+* [Algorithmia Support Portal](https://support.algorithmia.com/) includes FAQs and commonly requested information. It is also the portal for you to reach out for platform and administrative support or user-, data science-, and model-related questions.
+* Send us an email support request to [support@algorithmia.com](mailto:support@algorithmia.com). Please use your corporate domain email addresses if emailing us directly.
+* If using the Algorithmia Teams or Enterprise user interface (UI), a Help widget is available on the lower right portion of the screen.
```

Comment:
Small nit, the help widget only appears on teams if the user has a pro account.
```suggestion
* If using the Algorithmia Teams (with a Pro subscription) or Enterprise user interface (UI), a Help widget is available on the lower right portion of the screen.
```
cc @JamesHoover in case he'd like the text changed slightly

Reply:
Actually, just noticed this is below the `Pro` header, so may be alright to leave as-is